### ISSUE TITLE
Windows: Error when processing filenames with non-English Unicode characters (such as "ñ" and "ç")

### ISSUE THREAD

**Issue by LukasThyWalls** [szTheory/exifcleaner#17]
Hi

When the file names or folder names contains letters used in Spanish (I tested with "ñ" and "ç"), exifcleaner doesn't find them and doesn't process any file.

If its failing with "ñ" and "ç", is quite possible it will doing the same with other characters used in other languages, likely to a character encoding issue somewhere.

Thanks.

**Comment by szTheory** [comment]
Thanks! I'll take a look.

**Comment by szTheory** [comment]
I just tested on a Mac with filenames `añ2.png` and `a3ç4.jpg` within a folder called `cooñin` and it worked. What platform are you on, I'm guessing Windows? Also could you provide a screenshot  of the error if one is showing, as well as copy/paste the error text if it's showing?

![screenshot](https://user-images.githubusercontent.com/28652/75635313-ed5ed700-5c0c-11ea-86c8-04c4a493db09.png)

**Comment by LukasThyWalls** [comment]
Yes, i forgot to point that out. I'm on Windows (i don't know if other
thing might be relevant, be in a NTFS drive maybe? It's the default file
system)

Also, there is no error i noticed. When i drop the file it appears in the
list, but the exif infos are always empty, not 0 or other number, is blank
and didn't do anything more.

If i can gather more info about the problem, tell me how.

**Comment by szTheory** [comment]
I was able to reproduce on Windows and found the error message in Developer Tools.

```
internal/process/task_queues.js:85 TypeError: Cannot read property '0' of null
    at webpack:/src/common/add_files.js:134
    at process.exifData (internal/process/task_queues.js:85)
    at webpack:/src/common/add_files.js:22
```

```
processTicksAndRejections @ internal/process/task_queues.js:85
internal/process/task_queues.js:85 TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at updateRowWithExif (webpack:/src/common/table.js:9)
    at webpack:/src/common/add_files.js:27
    at processTicksAndRejections (internal/process/task_queues.js:85)
```

![exifcleaner error](https://user-images.githubusercontent.com/28652/75699865-26d02a80-5c7f-11ea-84c7-835b16ace6b0.jpg)

**Comment by szTheory** [comment]
Apparently this is a known issue with ExifTool and Windows/UTF-8 filenames. From their website:

>In Windows, ExifTool will not process files with Unicode characters in the file name. This is due to an underlying lack of support for Unicode filenames in the Windows standard C I/O libraries. [This deficiency was addressed in ExifTool 9.79, and ExifTool now supports Windows Unicode file names with some exceptions. See the WINDOWS UNICODE FILE NAMES section of the application documentation for details.]

I tried adding the `-charset filename=UTF8` command line argument as recommended [here](https://exiftool.org/exiftool_pod.html#WINDOWS-UNICODE-FILE-NAMES) but it didn't resolve the issue when I tested on Windows. I've opened a WIP pull request with the changes I tried https://github.com/szTheory/exifcleaner/pull/21. Maybe someone else can help resolve this issue. It would require cloning the git repo, switching to that `hotfix/windows-utf8-filename-bug` branch , making changes and testing them on Windows with the `npm run dev` command.

**Comment by szTheory** [comment]
I've pinned this issue so that everyone can see it. I'm too busy to spend time on this right now, but if someone can identify a fix I will fold it into ExifCleaner so that all Windows users can benefit. Thank you.

**Comment by szTheory** [comment]
@LukasThyWalls Sorry for the delay, but I've finally fixed this in v3.3.1 https://github.com/szTheory/exifcleaner/releases/tag/v3.3.1. Please let me know if you encounter any further issues.

**Comment by LukasThyWalls** [comment]
I can confirm it works now. Thanks.
